src/store/thunks/movies.js

```diff
@@ -0,0 +1,42 @@
+import axios from "../../core/axiosHelper";
+
+export const loadMovies = () => async (dispatch) => {
+  //FETCH AXIOS
+  const allMoviesData = await axios.get("/movies?limit=12");
+  dispatch({
+    type: "FETCH_MOVIES",
```

Comment:
Consider following the "actionTypes" approach here as well.

Reply:
This has been resolved using the ACTION TYPES approach.